ansible_collections/arista/avd/plugins/plugin_utils/cv_client/deploy_to_cv/finalize_change_control_on_cv.py

```diff
@@ -70,14 +72,29 @@ async def finalize_change_control_on_cv(change_control: CVChangeControl, cv_clie
             change_control_id=change_control.id, timestamp=cv_change_control.change.time, description="Automatic approval by AVD"
         )
         change_control.final_state = "approved"
+        LOGGER.info("finalize_change_control_on_cv: %s", change_control)
 
     # If requested state is "approved" we are done.
     if change_control.requested_state == "approved":
         return
 
     await cv_client.start_change_control(change_control_id=change_control.id, description="Automatically started by AVD")
     change_control.final_state = "running"
+    LOGGER.info("finalize_change_control_on_cv: %s", change_control)
 
     # If requested state is "running" we are done.
     if change_control.requested_state == "running":
         return
+
+    cv_change_control = await cv_client.wait_for_change_control_complete(cc_id=change_control.id, state="COMPLETED")
```

Comment:
Let's rename the function since it can wait for other states.
```suggestion
    cv_change_control = await cv_client.wait_for_change_control_state(cc_id=change_control.id, state="completed")
```